### DISCUSSION TITLE
Not start blueman at login?

### DISCUSSION THREAD

**Discussion by HubKing** [blueman-project/blueman#2414] [Q&A] [answered]
The DE already has a BT icon. I only need blueman when I need advanced features that the DE's BT settings don't provide. So, whenever I login, the tray shows two BT icons.

![image](https://github.com/blueman-project/blueman/assets/10900489/24450072-f4d7-4576-b9cb-9b1a620b214e)

It seems that there is no way to disable blueman from auto-starting at login, is there? If not, how about adding an option?

**Comment by cschramm** [comment, marked as answer]
See #2102. blueman typically ships `/etc/xdg/autostart/blueman.desktop` and desktops used to have a management for that where you could disable specific entries without fiddling with config files, but it turned out that Gnome nowadays does not. We are looking for workarounds that we could implement on our side. systemd did not turn out to be the solution (see #2105), so the best possible approach seems to be https://github.com/blueman-project/blueman/issues/2102#issuecomment-1676767325.

**Reply by HubKing** [reply]
Well, the file was indeed in /etc/xdg/autostart in Arch Linux, and not just Gnome, Plasma does seem to have GUI to manage the items in there either. The "Autostart" in Plasma's System Settings does not show items in that directory. I had to physically remove the file and then blueman did not autostart.

 It's quite weird that famous DE's do not support GUI to manage this. If I remember correctly, I think XFCE had GUI for this. I searched the web and it can be possible by create a counterpart in the user's local directory ( https://unix.stackexchange.com/questions/669372/xfce4-session-and-startup-where-are-autostart-items-saved ; He happens to have explained it for blueman) but the DE should be provide a GUI for this, instead of the user manually searching /etc/xdg/autostart directory to know the file name and manually create a text file in a hidden directory...
